subgraphs/network/README.md

```diff
@@ -0,0 +1,326 @@
+# EVM L1 Subgraph
```

Comment:
Will be good to rename this to `Network Subgraph` as now it's grown past EVM (and L1) 😉